jbi/runner.py

```diff
@@ -9,7 +9,7 @@
 from jbi.environment import Settings
 from jbi.errors import ActionNotFoundError, IgnoreInvalidRequestError
 from jbi.models import Actions, BugzillaWebhookRequest, RunnerLogContext
-from jbi.services import get_bugzilla
+from jbi.services import bugzilla
```

Comment:
```suggestion
from jbi.services.bugzilla import get_client as get_bugzilla
```
in the sake of brevity?

(`get_client()` is also more explicit, so...)

Reply:
Yeah, if we wanted to include the `client` context in that import alias, I think it'd be
`get_bugzilla_client` vs `bugzilla.get_client`, so same # of characters